packages/frontend/editor-ui/src/components/ImportWorkflowUrlModal.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import { useI18n } from '@/composables/useI18n';
+import { useUIStore } from '@/stores/ui.store';
+import { nodeViewEventBus } from '@/event-bus';
+import { VALID_WORKFLOW_IMPORT_URL_REGEX, IMPORT_WORKFLOW_URL_MODAL_KEY } from '@/constants';
+
+const i18n = useI18n();
+const uiStore = useUIStore();
+
+const url = ref('');
+const isValid = ref(true);
+
+const closeModal = () => {
+	uiStore.closeModal(IMPORT_WORKFLOW_URL_MODAL_KEY);
+};
+
+const confirm = () => {
+	if (!url.value.match(VALID_WORKFLOW_IMPORT_URL_REGEX)) {
+		isValid.value = false;
+		return;
+	}
+	nodeViewEventBus.emit('importWorkflowUrl', { url: url.value });
+	uiStore.closeModal(IMPORT_WORKFLOW_URL_MODAL_KEY);
```

Comment:
Since we already have the `closeModal` method in the component, we can just call it here:
```suggestion
	closeModal();
```